lib/src/raster/tile_loader.dart

```diff
@@ -17,31 +19,46 @@ class TileLoader {
   final TranslatingTileProvider _provider;
   final StorageImageCache _imageCache;
   final TileOffset _tileOffset;
+  final int _concurrency;
```

Comment:
this class has too many responsibilities, but it's all I have energy for tonight